vm/errors/error.go

```diff
@@ -33,5 +33,6 @@ const (
 	WrongArgumentTypeFormat     = "Expect argument to be %s. got: %s"
 	CantYieldWithoutBlockFormat = "Can't yield without a block"
 	DividedByZero               = "Divided by 0"
+	InvalidGobyCode             = "Invalid Goby code"
```

Comment:
If the Goby code is invalid, then we should return parser error. We shouldn't have this kind of error inside VM